src/VolleyManagement.Backend/VolleyManagement.Services/TeamService.cs

```diff
@@ -87,24 +87,21 @@ public ICollection<Team> Get()
         /// Create a new team.
         /// </summary>
         /// <param name="teamToCreate">A Team to create.</param>
-        public void Create(Team teamToCreate)
+        public Team Create(CreateTeamDto teamToCreate)
```

Comment:
Can we create team with the same name?

Reply:
@sdiachen extra lines will be deleted. For UTs  we create another issue, because firstly we want to fix the logic of TeamAggregate.
Yes, we can create. Is it bad?

Reply:
Did we allow 2 teams with the same name before refactoring?

Reply:
no